projects/plugins/boost/changelog/add-wpcom-command-palette-loader

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: added
+Comment: I'm being forced to add a changelog after merging with trunk I'll investigate the reason.
```

Comment:
We already know the reason, can we note it here? Or maybe add a link to https://github.com/Automattic/jetpack/pull/35635/files#r1502238082?

Reply:
Done.